docs/reference/mapping/types/boolean.asciidoc

```diff
@@ -179,6 +179,10 @@ The following parameters are accepted by `boolean` fields:
     enabled can still be queried using term or range-based queries,
     albeit slower.
 
+<<ignore-malformed, `ignore_malformed`>>::
+
+    Trying to index the wrong data type into a field throws an exception by default, and rejects the whole document. If this parameter is set to true, it allows the exception to be ignored. The malformed field is not indexed, but other fields in the document are processed normally. Accepts `true` or `false`.
```

Comment:
Can we base this text on some similar type like e.g. the numeric types described in `numeric.asciidoc`? For example, the incompatibility with the `script` parameter isn't mentioned here.

Reply:
Looking into this.

Reply:
Added incompatibility of `script` parameter.

Reply:
works for me.